release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -832,6 +845,10 @@ Support for empty files using the `--registry-config` and `--to` flags in `oc re
 [id="ocp-4-10-removed-features"]
 === Removed features
 
+{product-title} 4.10 removes the Jenkins Operator, which was a Technology Preview feature, from the OperatorHub page in the {product-title} web console interface. Bug fixes and support are provided through the end of a future release. Then, upgrades for the Jenkins Operator will no longer be available, and the Operator will not be supported.
```

Comment:
I think we have already produced our final version of the Jenkins operator. `0.7.2-28`.

cc @akram @jkhelil